### ISSUE TITLE
How to get the amount of transaction of type 'transfer_all'?

### ISSUE THREAD

**Issue by moderatelyOldMan** [polkadot-js/docs#292]
Hey, I've been struggling with finding a way to get the amount of transaction that was executed with "Transfer the full account balance, reap the sender" option. I use the RPC API without polkadot-js lib, and basically for every block I just make a "chain_getBlock" call, get the raw extrinsics from it and then decode them to JSON via TypeRegistry.createType('Extrinsic', tx).toHuman() and there I can get transaction method, sender\receiver and the value. However, in 'transferAll' transactions, there is no tx.method.args.value field, and there doesn't seem to be any endpoint in RPC API that allows to get that value. Am I missing something?

**Comment by jacogr** [comment]
The transaction emits events, see for instance this block on Kusama - https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.public.curie.radiumblock.co%2Fws#/explorer/query/12235400

The `balances.Transfer` event has the recipient & values. (Same as with a normal transfer)

**Comment by moderatelyOldMan** [comment]
Is there a way to get events for block/extrinsic via an HTTP request to API? Seems to be no endpoints related to 'event'.

**Comment by jacogr** [comment]
`api.query.system.events()`

**Comment by moderatelyOldMan** [comment]
I think we're not quite on the same page. You provided an interface name for this call, however, as I said in initial message, I don't use polkadot-js lib and send JSON-RPC requests. There is no RPC method name for such interface in the docs.

**Comment by moderatelyOldMan** [comment]
Yeah, I thought a lot about `getStorage`, but never found any keys to get the data. Thanks for the link, I'll ask there too. Cheers.

**Comment by jacogr** [comment]
Here is an example of constructing a storage key - https://substrate.stackexchange.com/questions/903/generating-a-storage-key-when-making-an-rpc-request
